src/Parsers/Kusto/KustoFunctions/KQLAggregationFunctions.cpp

```diff
@@ -27,19 +28,19 @@ bool ArgMax::convertImpl(String & out, IParser::Pos & pos)
 
     if (fn_name.empty())
         return false;
-    ++pos;
-    String expr_to_maximize = getConvertedArgument(fn_name, pos);
-    while (pos->type == TokenType::Comma)
+
+    const Interval & argument_count_interval = {2, Interval::max_bound};
+    const auto args = getArguments(fn_name, pos, ArgumentState::Parsed, argument_count_interval);
+    
+    for (std::string expr_to_return : args | std::views::drop(1))
```

Comment:
You could just go with:
```suggestion
    for (const auto& expr_to_return : args | std::views::drop(1))
```